projects/notebooks/testing/config.yaml

```diff
@@ -283,7 +285,7 @@ tests:
     artifacts_collected: no-screenshot-except-failed-and-zero
     # Should be true to capture Prometheus databases and properly generate the cluster usage metrics
     # Can be overriden by CI_ARTIFACTS_CAPTURE_PROM_DB
-    capture_prom_db: true
+    capture_prom_db: false
```

Comment:
you need to revert this modification (and maybe include it in the `dashboard_scale_test` preset)